src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -493,17 +493,39 @@ internal static string TupleMemberName(int position)
             return "Item" + position;
         }
 
+        private static bool IsElementNameForbidden(string name)
+        {
+            switch (name)
+            {
+                case "CombineHashCodes":
```

Comment:
This approach appears fragile. The less knowledge of the contents of well-known types that we need to bake into the compiler, the better. We should instead perform lookup in the underlying type, or alternately build this table at runtime by gathering member names.